pkgs/by-name/ss/sstp/package.nix

```
@@ -34,14 +36,20 @@ stdenv.mkDerivation {
   nativeBuildInputs = [
     pkg-config
     autoreconfHook
+    makeWrapper
   ];
 
   buildInputs = [
     libevent
     openssl
     ppp
+    cacert
   ];
 
+  postInstall = ''
+    wrapProgram $out/sbin/sstpc --add-flags "--ca-cert ${cacert}/etc/ssl/certs/ca-bundle.crt"
```

```suggestion
    wrapProgram $out/bin/sstpc --add-flags "--ca-cert ${cacert}/etc/ssl/certs/ca-bundle.crt"
```
Nix doesn't use `sbin` at all. In fact, every time a program generates files in `sbin`, they're copied to `bin` and `sbin` becomes a symbolic link to `bin`.

This is because we do not follow the Filesystem Hierarchy Standard: under that standard, binaries in `sbin` are SUID binaries or are only meant to be ran as the `root` user. Since everything in the `/nix/store` can only have `444` or `555` permissions, this doesn't apply to us.